digital_asset_types/src/rpc/filter.rs

```diff
@@ -11,7 +11,7 @@ pub struct AssetSorting {
 impl Default for AssetSorting {
     fn default() -> AssetSorting {
         AssetSorting {
-            sort_by: AssetSortBy::Created,
+            sort_by: AssetSortBy::Id,
```

Comment:
This may affect callers of the API because expecting created sort and switched to id.

Reply:
Good point. This was missed in an earlier Helius PR, we've been sorting by ID for a while now. You need to default to ID for people who use keyset pagination.

Another benefit: Created is the DB timestamp. Every RPC provider will have different values for created, meaning customers will have inconsistent responses between providers. ID is deterministic.